Hoje fiquei responsável pelo arquivo 'testes/comando_ver_minhas_compras_TST.py'.

O arquivo já existia, mas apenas a base. Para completá-lo, consuletei os arquivos 'comando_ver_compra_TST.py' e 'comando_solicitar_pag_buscar_usuarios_TST.py'.
Considerando a documentação presente em 'comando_ver_minhas_compras.py', há 3 cenários a serem validados: sessão nula, sessão válida e sessão admin. Então criei esses 3 casos de teste.

Para o primeiro (ses == None), como a validação é por 'assert', precisei encapsulá-lo em um try/except para AssertError, pois não continuava a execução dos testes. Para esse, foi gerado uma página html com texto puro dizendo apenas que teve erro. 
Apesar de não estar errado, do ponto de vista da usabilidade, talvez fosse melhor gerar uma página de erro (html_pag_mensagem_de_erro) ao invés de simplesmente quebrar com um assert.

Para a sessão comum, a página de minhas compras foi gerada devidamente, com uma lista com 2 compras associadas àquele usuário.

Para a sessão admin, foi gerada uma página estruturada da lista "Minhas Compras", mas estava vazia. Não sei qual deveria ser o comportamento aqui (AssertError, pagina de erro ou essa lista vazia mesmo), mas a documentação indica que essa página não deveria ser gerada para usuários admin. Acredito que a lista só veio vazia porque o usuário não tem compras associadas a ele, não por ser admin.
